projects/plugins/boost/app/assets/src/js/pages/settings/sections/History.svelte

```diff
@@ -74,6 +75,8 @@
 	<ReactComponent
 		this={PerformanceHistory}
 		onToggle={onToggleHistory}
+		isFreshStart={$dismissedAlerts.performance_history_fresh_start !== true}
+		onDismissFreshStart={() => ( $dismissedAlerts.performance_history_fresh_start = true )}
```

Comment:
Can you bubble these up to the parent component?

You can see how I've done it here - https://github.com/Automattic/jetpack/pull/33278/files#diff-0c8a77ded084513471e3c3be7ce4faf020dc88955b75d71c02d29405b7e703ffL7-L8

Reply:
Updated https://github.com/Automattic/jetpack/pull/33133/commits/a92a7031f1ebcdeec564cf2d735b37256811fe6d